www-servers/lighttpd/lighttpd-1.4.64-r1.ebuild

```
@@ -0,0 +1,236 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} )
+
+inherit autotools lua-single readme.gentoo-r1 systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Lightweight high-performance web server"
+HOMEPAGE="https://www.lighttpd.net https://github.com/lighttpd"
+SRC_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/${P}.tar.xz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="brotli dbi doc gnutls ipv6 kerberos ldap lua maxminddb mbedtls mmap mysql nettle nss pcre php postgres rrdtool sasl selinux ssl sqlite system-xxhash test unwind webdav xattr zlib zstd"
```

`metadata.xml` says this is recommended, so we should turn it [on by default](https://devmanual.gentoo.org/general-concepts/use-flags/index.html#iuse-defaults):
```suggestion
IUSE="+brotli dbi doc gnutls ipv6 kerberos ldap lua maxminddb mbedtls mmap mysql nettle nss pcre php postgres rrdtool sasl selinux ssl sqlite system-xxhash test unwind webdav xattr zlib zstd"
```

Anything else we should do that for?

---

Probably all of: pcre lua nettle ssl zlib